doc/cookbook/source/examples/classifier/id3.rst

```diff
@@ -0,0 +1,37 @@
+====================
+ID3 Classifier
+====================
+
+ID3 classifier is a decision tree based learning algorithm that can perform multi-class classification. The ID3 algorithm begins with the original set :math:`{\bf S}` as the root node. On each iteration of the algorithm, it iterates through every unused attribute of the set :math:`{\bf S}` and calculates the information gain :math:`{ IG(\bf S)}` of that attribute. It then selects the attribute which has the largest information gain value. The set :math:`{\bf S}` is then split by the selected attribute. The algorithm continues to recurse on each subset till a stopping criteria is reached. 
```

Comment:
Not sure we need the math S here, you never really use it. So just talk about the training set rather than using math S